### PR TITLE
Fix nullability and epoch

### PR DESCRIPTION
**Pull request by erwan-joly**

### CODE REVIEW

File: src/Fynance.Tests/TickerTests.cs [hunk 1]
```diff
@@ -55,6 +56,38 @@ public void Should_download_ticker_data_from_yahoo_finance_by_start_finish_date(
...
+            Assert.IsNotNull(ticker.Result);
+            Assert.IsNotNull(result);
+
+            Assert.AreEqual(result.Symbol, symbol);
+
+            Assert.IsNotNull(result.Quotes);
+            Assert.IsTrue(result.Quotes.Length > 0);
+
+            Assert.IsNotNull(result.Dividends);
+        }
```
Comment (felipeoriani) [inline src/Fynance.Tests/TickerTests.cs]: Fix the tabulations.

File: src/Fynance/Yahoo/YUtils.cs [hunk 2]
```diff
@@ -114,11 +114,15 @@ public static FyQuote[] GetQuotes(this YResultResponse resultResponse, TimeZoneI
 			var ohlc = resultResponse.Indicators.Quote.First();
 
 			var timestamps = resultResponse.TimeStamp;
+            var quotes = new List<FyQuote>();
+			if (timestamps == null)
+                return quotes.ToArray();
```
Comment (felipeoriani) [inline src/Fynance/Yahoo/YUtils.cs]: Can you fix the tabulations?

**Review by felipeoriani — CHANGES_REQUESTED** [review]
Thank you for your contribution Erwan.
Notice small tabulation issues on the code and I have commented on it.
In Visual Studio (and Visual Studio Code) you can use Ctrl+R+W to toogle tab/space chars and easily fix it.

File: src/Fynance/Yahoo/YUtils.cs (hunk 2 above)
Comment (0Lucifer0) [reply src/Fynance/Yahoo/YUtils.cs]: done

Comment (erwan-joly) [reply src/Fynance/Yahoo/YUtils.cs]: done

**Comment by erwan-joly** [comment]
Small note for consideration the normal default for tabbing in C# on most of IDE is 4 spaces. Visual studio for exemple replace automatically tab by space by default
image

**Comment by felipeoriani** [comment]
Thank you for your contribution Erwan!

**Pull request merged**